### ISSUE TITLE
SaveSuiteAction serialize CSV profile properties incorrectly

### ISSUE THREAD

**Issue by TheAutomatingMrLynch** [Seddryck/NBi#612]
SaveSuiteAction is not correctly serializing `EmptyCell`, `MissingCell` and `FirstRowHeader` properties of `CsvProfileXml` when they differ from their default value. 

The issue seems to be in `NBi.Xml.XmlManager.Persist()` as the values are correct in the `TestSuiteXml` object all the way to the `serializer.Serialize(writer, testSuite)` part. 

In all three cases the resulting file looks like this:
```
<?xml version="1.0" encoding="utf-8"?>
<testSuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://NBi/TestSuite">
  <settings />
</testSuite>
```

BUT if you include another property like `FieldSeparator` the file includes both `FieldSeparator` and `FirstRowHeader`:
```
<?xml version="1.0" encoding="utf-8"?>
<testSuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://NBi/TestSuite">
  <settings>
    <csv-profile field-separator="#" first-row-header="true" />
  </settings>
</testSuite>
```

**Comment by TheAutomatingMrLynch** [comment]
Found the solution! The three properties in question were not included in NBi.Xml.Settings.SettingsXml.ShouldSerializeCsvProfile(). Pull request on the way.

**Comment by Seddryck** [comment]
Will check this as soon as the build is working again.

**Comment by Seddryck** [comment]
Fix available in [1.23.0-beta0056](https://ci.appveyor.com/project/Seddryck/nbi/builds/36814366)
